tools/generator/src/PBXTargets+Extensions.swift

```diff
@@ -0,0 +1,7 @@
+import XcodeProj
+
+public extension Dictionary where Value: PBXTarget {
```

Comment:
We use it in `setTargetDependencies`, as well. So, I made it public.

Reply:
But `internal`, the default, is visible there.

Reply:
Just got back to my computer and tried without the `public`. It works. I thought that it had to be `public` because the extended class was not in the module. I will remove it.